style.css

```diff
@@ -601,11 +600,13 @@ input:focus:invalid {
  #container1 {
     height: 100vh;
     background-color: #1c1a19;
-    background-image: url("images/Illustration-header.png");
+    background-image: url(images/Illustration-header.png);
     background-repeat: no-repeat;
     background-position: right;
+    background-position-y: 0.2rem;
     background-size: 40%;
     padding-bottom: 3rem;
```

Comment:
- [ ] As requested by previous reviewer you should stick to the design as much as possible. Please adjus the spacing and and there should be gap between each cards. Please see image below for reference.
### Your design
![aa](https://github.com/Tshepo45/Portfolio-desktop-version/assets/81727988/a150571b-db6b-48a9-9fa4-01be71d1db6e)

### Figma design
![his](https://github.com/Tshepo45/Portfolio-desktop-version/assets/81727988/2d0346c8-0b3c-4189-b077-dc421ef66aa8)